LocationSimulator/ViewController/MapViewController/MapViewController+LocationSpooferDelegate.swift

```diff
@@ -119,7 +118,6 @@ extension MapViewController: LocationSpooferDelegate {
 
         // Update the current application status.
         NotificationCenter.default.post(name: .StatusChanged, object: self, userInfo: [
-            "device": spoofer.device,
```

Comment:
userInfo["device"] wasn't used in the observer, so to simplify things I just removed this.